hts.py

```diff
@@ -35,7 +35,7 @@ def start(config_path):
     run_dir = os.path.join(htsohm_dir, run_id)
     os.makedirs(run_dir, exist_ok=True)
     file_name = os.path.join(run_dir, 'config.yaml')
-    with open(file_name, 'w') config_file:
+    with open(file_name, 'w') as config_file:
```

Comment:
run-time errors should generally be squashed in to wherever the original code was.